zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -989,18 +1055,20 @@ private void renameFolder(NotebookSocket conn, HashSet<String> userAndRoles,
     }
   }
 
-  private boolean isCronUpdated(Map<String, Object> configA, Map<String, Object> configB) {
-    boolean cronUpdated = false;
-    if (configA.get("cron") != null && configB.get("cron") != null && configA.get("cron")
-        .equals(configB.get("cron"))) {
-      cronUpdated = true;
-    } else if (configA.get("cron") == null && configB.get("cron") == null) {
-      cronUpdated = false;
-    } else if (configA.get("cron") != null || configB.get("cron") != null) {
-      cronUpdated = true;
+  private boolean isNoteConfigValueUpdated(String key,
+                                           Map<String, Object> configA,
+                                           Map<String, Object> configB) {
+    boolean updated = false;
+    if (configA.get(key) != null && configB.get(key) != null && configA.get(key)
+        .equals(configB.get(key))) {
```

Comment:
a bit hard to read - put `configA.get(key)` before `.equals(configB.get(key))`

Reply:
@felixcheung This was fixed at https://github.com/apache/zeppelin/pull/2671/commits/f4f53c5e9d65a31d57cadc86820a96f385cedee7. Thanks.